src/cavegen.cpp

```diff
@@ -23,46 +23,174 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "mapgen_v5.h"
 #include "mapgen_v6.h"
 #include "mapgen_v7.h"
+#include "mg_biome.h"
 #include "cavegen.h"
 
 NoiseParams nparams_caveliquids(0, 1, v3f(150.0, 150.0, 150.0), 776, 3, 0.6, 2.0);
 
 
-///////////////////////////////////////// Caves V5
+////
+//// CavesNoiseIntersection
+////
 
+CavesNoiseIntersection::CavesNoiseIntersection(
+	INodeDefManager *nodedef, BiomeManager *biomemgr, v3s16 chunksize,
+	NoiseParams *np_cave1, NoiseParams *np_cave2, int seed, float cave_width)
```

Comment:
You mean for the seed?

Zeno pointed this out as well.  I'm going to fix the seed type issue all at once, in a separate commit because it touches on noise.cpp quite a bit.